Docs/Whats_New.md

```diff
@@ -18,6 +18,8 @@ MoltenVK 1.2.8
 
 Released TBD
 
+- Add `MVK_USE_METAL_PRIVATE_API` build setting to allow **MoltenVK** to be built with access to _Metal_ private API calls.
```

Comment:
This may be bikeshedding, but Apple typically uses the IBM jargon "Service Provider Interface" (SPI) for a private interface like this.

Reply:
Fair enough. Although AFAICT, colloquially, everyone seems to refer to them as "private API's", so I think we should continue to include use of that terminology. Although, when you do your add, feel free to add something like an "(aka SPI calls)" into these comments if you feel SPI is well known.